ql/src/test/org/apache/hadoop/hive/ql/lockmgr/TestDbTxnManager2.java

```diff
@@ -2429,7 +2433,7 @@ private void testConcurrent2MergeUpdatesConflict(boolean slowCompile) throws Exc
     List<String> res = new ArrayList<>();
     driver.getFetchTask().fetch(res);
     Assert.assertEquals(2, res.size());
-    Assert.assertEquals("Lost Update", "[earl\t10, amy\t10]", res.toString());
+    Assert.assertTrue("Lost Update", isEqualCollection(res, asList("earl\t10", "amy\t10")));
```

Comment:
nit: Why not `Assert.assertArrayEquals`?

Reply:
it uses Arrays.deepEquals that is order sensitive, unlike the isEqualCollection